prac_10/testing.py

```diff
@@ -0,0 +1,84 @@
+"""
+CP1404/CP5632 Practical
+Testing demo using assert and doctest
+"""
+
+import doctest
+from prac_07.car import Car
+
+
+def repeat_string(s, n):
+    """Repeat string s, n times, with spaces in between."""
+    strings = []
+    while n > 0:
```

Comment:
This can be done much more easily... see the solution.

Reply:
Why use a `while` loop instead of a `for` loop? How is this *indefinite iteration*?